xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd

```diff
@@ -176,7 +176,7 @@ Rail transport, Roads and Road transport
 	<!-- ======================================================================= -->
 	<xsd:element name="ScheduledStopPoint" abstract="false" substitutionGroup="Point">
 		<xsd:annotation>
-			<xsd:documentation> A POINT where passengers can board or alight from vehicles.</xsd:documentation>
+			<xsd:documentation> A POINT where passengers can board or alight from vehicles. It is open, which hiearchical level such a point has. It can represent a single door (BoardingPosition) or a whole ZONE. The association to the physical model is done with STOP ASSIGNMENTs.</xsd:documentation>
```

Comment:
```suggestion
			<xsd:documentation> A POINT where passengers can board or alight from vehicles. It is open, which hierarchical level such a point has. It can represent a single door (BoardingPosition) or a whole ZONE. The association to the physical model is done with STOP ASSIGNMENTs.</xsd:documentation>
```